js/exercise-00.js

```diff
@@ -0,0 +1,20 @@
+/* 
+   Create a constant variable of each of the following primitive types:
+   - String
+   - Number
+   - Boolean
+   - Null
+   - Undefined
+
+   Log out each variable to the terminal at the end.
+*/
+const string = "Manel";
+const number = 18;
+const boolean = true;
+const nulo = null;
```

Comment:
Use 🇬🇧 to name variables.

```suggestion
const nullValue = null;
```

FIX: name properly variable